docs/cloud/workspaces.md

```diff
@@ -45,22 +45,33 @@ Click **Add**.  The user will receive an email invitation to join the organizati
 
 To remove a user from the organization, select the options menu button (hamburger) to the right of the user and click **Remove**.
 
-## Rebooting and Disabling Workspaces
+## Workspace Sleeping
 
-Issues with workspaces are uncommon, but if you ever need to reboot your workspace, you can do so from the **Settings** > **Advanced** tab.
-
-Under the **Workspace Status** section choose the `Reboot` option. This will cycle the workspace, which will cause a brief interruption in service. This typically takes no longer than 30 seconds.
+Steampipe Cloud will put workspaces to sleep if they have been inactive for 7 days.  When a workspace is sleeping the database is effectively shut down.  You will not be able to query the workspace via any means until you wake it again, and all pipelines will be disabled.
```

Comment:
@johnsmyth we were generally trying to avoid "put to sleep" to avoid any connotations.

```suggestion
Steampipe Cloud will put workspaces into sleep mode if they have been inactive for 7 days.  When a workspace is sleeping the database is effectively shut down.  You will not be able to query the workspace via any means until you wake it again, and all pipelines will be disabled.
```